utils/system-notification/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2021 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See https://www.gnu.org/licenses/gpl-2.0.txt for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=system-notification
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:=Florian Eckert <fe@dev.tdt.de>
+PKG_LICENSE:=GPL-2.0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/system-notification
+  SECTION:=base
+  CATEGORY:=Base system
+  TITLE:=System notification
+endef
+
+define Package/system-notification/description
+  This package provid a ubus backend with which system notification
+  can be stored.
+
+  The following ubus methods are available:
+  * add
+  * remove
+  * list
+  * flush
```

Comment:
there's no url listed, is this the only documentation?

Reply:
So far there is no other documentation. I think I will expand the information here.